src/main/resources/db/migration/common/V1_332__add_task_cleanup_current_event.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size)
+VALUES (nextval('aut_seq'),'CleanupCurrentEvent','Cleans up the event','0 * 0/4 * * *',true,50);
```

Comment:
can we provide a more detailed description of what this process is doing please?
e.g. 
"Cleans up the event is_current flag"